### ISSUE TITLE
some small visual problems

### ISSUE THREAD

**Issue by alexlib** [OpenPIV/openpiv_gui#27]
size of the values cell
![image](https://user-images.githubusercontent.com/747110/52150448-a6ed2080-2678-11e9-87b8-03815fbd62e6.png)

@royekorolik please take a look:
when I select one of the buttons, colors become white and invisible

![image](https://user-images.githubusercontent.com/747110/52150471-bd937780-2678-11e9-8517-edb81bcd8563.png)

**Comment by royekorolik** [comment]
is this still a problem?

**Comment by alexlib** [comment]
yes, on mac it is white - can you please change it so the choise does not change a color?
